deploy/gateway/values.yaml

```diff
@@ -198,7 +198,11 @@ metrics:
       #     (
       #       sum(rate(twingate_gateway_client_authentication_total{code!="200"}[5m])) /
       #       sum(rate(twingate_gateway_client_authentication_total[5m]))
-      #     ) > 0.1
+      #     ) > 0.1 
+      #     and
+      #     ( 
+      #       sum(rate(twingate_gateway_client_authentication_total[5m])) > 1
+      #     )
```

Comment:
It's an edge case: during low traffic or a cold start, if the first request is an error, the percentage would be 100% which would fire an alert. This guard clause `sum(rate(twingate_gateway_client_authentication_total[5m])) > 1` (300 requests across 5 minutes) should allow sufficient data points for us to make the calculation. @minhtule WDYT?

Reply:
Yes, I think we need to handle this scenario because Gateway is not Internet facing service so low traffic or no traffic periods are expected.